tests/functional/gcs/fixture_data/role_list.json

```
@@ -0,0 +1,25 @@
+{
+    "DATA_TYPE": "result#1.0.0",
+    "code": "success",
+    "data": [
+        {
+            "DATA_TYPE": "role#1.0.0",
+            "id": "{role_id_1}",
+            "collection": null,
+            "principal": "rn:globus:auth:identity:{user_id_1}",
+            "role": "owner"
+        },
+        {
+            "DATA_TYPE": "role#1.0.0",
+            "id": "{role_id_2}",
+            "collection": "{collection_id_1}",
+            "principal": "rn:globus:auth:group:{group_id_1}",
```

```suggestion
            "principal": "urn:globus:groups:id:{group_id_1}",
```

The format for a Group URN is `urn:globus:groups:id:`.
As above, our tests won't pass or fail on the basis of this, but it's good to be consistent with the real data format.